components/azure-iot/jenkins/esp32_c.sh

```diff
@@ -0,0 +1,45 @@
+echo "Cross-compiling a non-working ESP32 example"
```

Comment:
Is it still planned that Microsoft will be using this CI build file? If not, it can be removed — we don't use Jenkins and all examples will be built by an existing CI system.

Reply:
(same for other files under jenkins/ directory)